{"name": "test_Login_with_Invalid_Password", "status": "failed", "statusDetails": {"message": "AssertionError: False != True", "trace": "self = <Test_Cases.test_LoginPage.Github_Login_Functionality_validation testMethod=test_Login_with_Invalid_Password>\n\n    def test_Login_with_Invalid_Password(self):\n        self.driver.get(self.baseURL)\n        parameter1 = str(config['sign_in']['email'])\n        parameter2 = str(config['sign_in']['invalid_password'])\n        self.lp.UserLogin(parameter1, parameter2)\n        time.sleep(5)\n        result = self.lp.VerifyLogin()\n>       self.assertEqual(result, True)\nE       AssertionError: False != True\n\ntest_LoginPage.py:88: AssertionError"}, "attachments": [{"name": "log", "source": "a0bd6a82-f4a0-4db3-8679-e42fac3914f1-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "20716812-99e2-4bfa-9830-1997ed50a3a4-attachment.txt", "type": "text/plain"}], "start": 1559837102363, "stop": 1559837146509, "uuid": "943c573b-ea24-4220-9bdb-ae1490ffd2b6", "historyId": "6119552fbc1b2e74340718249d6dc8de", "fullName": "test_LoginPage.Github_Login_Functionality_validation#test_Login_with_Invalid_Password", "labels": [{"name": "suite", "value": "test_LoginPage"}, {"name": "subSuite", "value": "Github_Login_Functionality_validation"}, {"name": "host", "value": "WKMIN7880754"}, {"name": "thread", "value": "36785-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_LoginPage"}]}